docs/release/trg-5/trg-5-2.md

```diff
@@ -104,6 +104,8 @@ The file **must** contain the [_Apache 2.0 Licence_](https://github.com/catenax-
 The README.md file **must** contain:
 
 - a brief explanation of what the Helm chart will install
+- a section of "Prerequisites" includes the required kubernetes version, helm version and other necessary prerequisites and their versions. See an example [here](https://github.com/bitnami/charts/tree/main/bitnami/postgresql#prerequisites).
+- a section of "TL;DR" that includes the helm commands to add anf install the helm chart, see example [here](https://github.com/bitnami/charts/tree/main/bitnami/postgresql#tldr)
```

Comment:
Not convinced that teams will know what to add (→ URL of Helm repo), also there is a typo.

Reply:
In our TRG we should be more precise in what we expect. Examples in external repos might change without our knowledge.

Reply:
changed to inline examples